lib/src/element/status_bar.cpp

```diff
@@ -92,6 +92,11 @@ namespace cycfi::elements
       _start_pos = clamp(val, 0.0, 1.0);
    }
 
+   void busy_bar_base::animation_width(double val)
```

Comment:
I've also been mulling about using the "styler" concept (see the buttons, for example) as a more adaptive design strategy, but that can come later.